Doc/extending/extending.rst

```diff
@@ -127,10 +127,10 @@ Intermezzo: Errors and Exceptions
 
 An important convention throughout the Python interpreter is the following: when
 a function fails, it should set an exception condition and return an error value
-(usually a ``NULL`` pointer).  Exceptions are stored in a static global variable
-inside the interpreter; if this variable is ``NULL`` no exception has occurred.  A
-second global variable stores the "associated value" of the exception (the
-second argument to :keyword:`raise`).  A third variable contains the stack
+(usually a ``NULL`` pointer).  Exception information is stored in 3 static
+global variables inside the interpreter.  The first is the exception type,
```

Comment:
Maybe   "3 static global variables inside the interpreter"  --> "3 fields on the interpreter's thread state" ?

Reply:
Done, using 'members'.

Reply:
I concur with @iritkatriel, they are not global variables. For example, you can two interpreters, each one has its own exception information.

@terryjreedy wrote "done", but the current PR stil says "3 static global variables".